### PR TITLE
Upgrade to DB Writer common 3.0.1

### PR DESCRIPTION
**Pull request by ErikZigo**

### CODE REVIEW

**Comment by MiroCillik** [comment]
LGTM
Nezabudni, ked to budes nasadzovat prepnut v Developer portaly yaml na json ;)

**Comment by ErikZigo** [comment]
Jo to vim :D

File: src/Keboola/DbWriter/Snowflake/Application.php [hunk 1]
```diff
@@ -30,7 +31,11 @@ public function runAction()
             if ($table['incremental']) {
                 $table['dbName'] = $writer->generateTmpName($table['dbName']);
             }
-            $table['items'] = $this->reorderColumns($manifest['columns'], $table['items']);
+
+            $table['items'] = $this->reorderColumns(
```
Comment (pivnicek) [inline src/Keboola/DbWriter/Snowflake/Application.php]: tohle je definovanej v common?

**Comment by pivnicek** [comment]
aha, miro rekl stejne lol :-)

File: src/Keboola/DbWriter/Snowflake/Application.php (hunk 1 above)
Comment (ErikZigo) [reply src/Keboola/DbWriter/Snowflake/Application.php]: Snowflake writer ma staging v S3, proto to ma tady vlastni implementaci.

```
        "staging_storage": {
          "input": "s3"
        },
```

**Pull request merged**